data.js

```diff
@@ -43,5 +43,14 @@ const peopleChoiceData = [
         favMovieID: '284054-black-panther',
         favSeries: 'Big Bang Theory',
         favSeriesID: '1418-the-big-bang-theory'
+    },
+    {
+        username: 'GettingWeirdKnowledge',
+        favAnime: 'Your Name',
+        favAnimeID: '372058',
```

Comment:
Hey, can you make it an anime series? The way this project is designed requires an anime series name.